e2e-tests/gef/cypress/e2e/facility/pre-submission/facility-end-date-enabled/about-facility.spec.js

```diff
@@ -111,22 +108,20 @@ context('About Facility Page  - feature flag enabled', () => {
       cy.visit(relative(`/gef/application-details/${application.id}/facilities/${facilityId}/about-facility`));
       cy.keyboardInput(aboutFacility.facilityName(), 'Name');
       aboutFacility.shouldCoverStartOnSubmissionNo().click();
-      cy.keyboardInput(aboutFacility.coverStartDateDay(), `${now.getDate()}-`);
-      cy.keyboardInput(aboutFacility.coverStartDateMonth(), now.getMonth() + 1);
-      cy.keyboardInput(aboutFacility.coverStartDateYear(), now.getFullYear());
-      cy.keyboardInput(aboutFacility.coverEndDateDay(), `${now.getDate()}-`);
-      cy.keyboardInput(aboutFacility.coverEndDateMonth(), now.getMonth() + 1);
-      cy.keyboardInput(aboutFacility.coverEndDateYear(), now.getFullYear());
+
+      cy.completeDateFormFields({ idPrefix: 'cover-start-date', day: `${todayDay}-`, month: `${todayMonth} + 1`, year: todayYear });
```

Comment:
I think this test is failing because the `+ 1` is outside the ${todayMonth} so is literally in the string - however `todayMonth` from dateConstants already takes in the +1 factor, so this should just be: 

`cy.completeDateFormFields({ idPrefix: 'cover-start-date', day: `${todayDay}-`, month: todayMonth, year: todayYear });` (same below)

Reply:
Whoops thank you